stance_detection.py

```diff
@@ -87,44 +113,51 @@ def _threshold_parser(self, val, threshold_ranges):
         return numbuckets
 
     def _word_tokenize(self, str_list):
-        return map(lambda s: nltk.word_tokenize(s), str_list)
+        return list(map(lambda s: nltk.word_tokenize(s), str_list))
```

Comment:
I assume you're casting to `list` here because you're using python 3? I've been using python 2 because I already had all of the NLP libraries (with all of the large downloads as well) in python 2 for the assignments. 

Sorry, I should have added that to the README.

Are you cool with using Python 2 as well?